packages/rum_sdk/lib/src/rum_navigation_observer.dart

```diff
@@ -1,32 +1,34 @@
 import 'package:flutter/widgets.dart';
 import 'package:rum_sdk/rum_sdk.dart';
 
-class RumNavigationObserver extends RouteObserver<PageRoute<dynamic>>{
-@override
+class RumNavigationObserver extends RouteObserver<PageRoute<dynamic>> {
+  @override
   void didPop(Route route, Route? previousRoute) {
-  super.didPop(route, previousRoute);
-  RumFlutter().setViewMeta(name:previousRoute?.settings.name);
-  RumFlutter().pushEvent("view_changed",attributes: {
-    "route":previousRoute?.settings.name
-  });
+    super.didPop(route, previousRoute);
+    RumFlutter().setViewMeta(name: previousRoute?.settings.name);
+    RumFlutter().pushEvent("view_changed", attributes: {
+      "fromView": route.settings.name,
+      "toView": previousRoute?.settings.name,
```

Comment:
According to the Faro-Frontend team, this is the correct format for the `view_changed` event